data/schema/core/addons.cfg

```diff
@@ -26,6 +26,7 @@
 		[/then]
 	[/if]
 	{DEFAULT_KEY no_leader bool no}
+	{DEFAULT_KEY canrecruit bool yes}
```

Comment:
Yeah, the side had both no_leader=yes and canrecruit=yes.

Reply:
I think in that case you can just remove the `canrecruit=yes` since it's not doing anything... especially if there's also no `type` key present. Alternatively, you might want to remove the `no_leader=yes` key and replace it with faction lock.

Reply:
I'll remove `canrecruit=yes`.